### DISCUSSION TITLE
furniture nodes

### DISCUSSION THREAD

**Discussion by W4yv3rn** [ErinaSugino/Starbound-SBR-Mod#79] [Trouble Shooting] [answered]
i dont know if im missing something or what, but i cant seem to get npcs to use these furniture nodes and the only time i seem to interact with the npcs is with the aphrodites bow.

**Comment by ErinaSugino** [comment, marked as answer]
By default their natural arousal gain is set to take 3-7 in-game days. This time only ticks when the NPC is loaded in, which requires being in the same world instance and near them.
